code/src/pages/cart/CartPage.tsx

```diff
@@ -19,118 +22,127 @@ type CartItemsType = {
-              />
-              <AppTypography
-                component="p"
-                className="spa-order-summary__text"
-                variant="subtitle2"
-              >
-                {formatPrice(subtotalItems)}
-              </AppTypography>
-            </AppBox>
-            <AppBox className="spa-order-summary__row">
-              <AppTypography component="p" translationKey="delivery.label" />
-              <AppTypography component="p" translationKey="free.label" />
-            </AppBox>
+      </PageWrapper>
+    );
+  }
+
+  const cartItemsBlock = cartItems.items.map((item: CartItem) => (
+    <CartItem key={item.productId} item={item} />
+  ));
+
+  const totalPrice = formatPrice(cartItems.totalPrice ?? 0);
+
+  return (
+    <PageWrapper>
+      <AppBox className="spa-cart-page">
+        <AppBox className="spa-cart-page__content">
+          <AppBox className="spa-cart-page__items">
             <AppTypography
-              component="p"
-              className="spa-order-summary__underline-text"
-              translationKey="country.label"
+              className="spa-cart-page__items--label"
+              variant="h3"
+              component="h1"
+              translationKey="myCart.label"
+              data-testid="myCartLabel"
             />
-            <AppBox className="spa-order-summary__row spa-order-summary__total-line">
+            {cartItemsBlock}
+          </AppBox>
+          <AppBox className="spa-cart-page__order-summary">
+            <AppTypography
+              className="spa-cart-page__order-summary--label"
+              variant="h1"
+              component="h1"
+              translationKey="orderSummary.label"
+              data-testid="orderSummaryLabel"
+            />
+            <AppBox className="spa-order-summary__details">
+              <AppBox className="spa-order-summary__row">
+                <AppTypography
+                  component="p"
+                  className="spa-order-summary__text"
+                  translationKey="subtotal.label"
+                  data-testid="subtotalLabel"
+                />
+                <AppTypography
+                  component="p"
+                  className="spa-order-summary__text"
+                  variant="subtitle2"
+                >
+                  {totalPrice}
+                </AppTypography>
+              </AppBox>
+              <AppBox className="spa-order-summary__row">
+                <AppTypography
+                  component="p"
+                  translationKey="delivery.label"
+                  data-testid="deliveryLabel"
+                />
+                <AppTypography
+                  component="p"
+                  translationKey="free.label"
+                  data-testid="freeLabel"
+                />
+              </AppBox>
               <AppTypography
-                className="spa-order-summary__total"
-                variant="subtitle2"
                 component="p"
```

Comment:
Same

Reply:
> Same

What do you want ? helper-function ?? For what ?

Reply:
You removed `component`. It is correct